css/_toastr.scss

```diff
@@ -93,7 +93,7 @@
 
 #toast-container.notification-bottom-right {
   bottom: 135px;
-  right: 13px;
+  right: 28px;
 }
```

Comment:
Propose to do this as the part of https://github.com/jitsi/jitsi-meet/pull/1045
I mean when we'll extract some CSS values to variables.